selfdrive/car/docs_definitions.py

```diff
@@ -68,6 +69,18 @@ class Harness(Enum):
   ford_q4 = "Ford Q4"
   none = "None"
 
+class HarnessKitContent(Enum):
+  obd_ii_kit = ["1 harness box", "1 harness connector", "1 comma power v2", "1 RJ45 cable (7 ft)"]
+  j533_kit = ["1 harness box", "1 harness connector", "1 long OBD-C cable", "1 USB-C coupler"]
+  nissan_kit = ["1 harness box", "1 harness connector", "1 RJ45 cable (7 ft)", "1 long OBD-C cable", "1 USB-C coupler"]
+  default = ["1 harness box", "1 harness connector", "1 comma power v2", "1 RJ45 cable (7 ft)"]
+
+HarnessExceptions : Dict[Enum, HarnessKitContent] = {
```

Comment:
each `CarInfo` should take an argument called something like `kit_content`, which is a list of parts